### ISSUE TITLE
Limit IP addresses to a single connection

### ISSUE THREAD

**Issue by caressingbeast** [caressingbeast/ecreaylist#17]
Since @lousando thinks it might be a good idea to open a new tab and log in as a new user, we should limit connections by the IP address.

**Comment by lousando** [comment]
Lmao

**Comment by lousando** [comment]
Auth token + fingerprint + retina scan

**Comment by caressingbeast** [comment]
@lousando Look, this is a simple open-source project...

**Comment by lousando** [comment]
I wonder if there's a better way, like to fingerprint a device. A simple will bypass this.

**Comment by caressingbeast** [comment]
@lousando Yeah, but I really don't care enough about one of the few users that's willing to go through a proxy just to do more upvotes/downvotes ha.
